linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -62,14 +62,14 @@ pub struct LocalKubernetesNet {
 #[cfg(any(test, feature = "test"))]
 impl SharedLocalKubernetesNetTestingConfig {
     pub fn new(network: Network, binaries: Option<Option<PathBuf>>) -> Self {
-        if binaries.is_none() {
+        if std::env::var("GITHUB_ACTIONS").unwrap_or_default() == "true" && binaries.is_none() {
```

Comment:
Is there no better way to inject the right value for `binaries` in CI?

Reply:
Yeah, this is not exactly pretty, but I couldn't find another way to do it :/ Do you know of any better ways? We could pass some flag from CI also, but not sure that's better

Reply:
Using an environment variable seems like the right thing to do but it should be a variable `LINERA_SOMETHING` that we define ourselves and with a clear meaning.

Reply:
So `GITHUB_ACTIONS` is an env var that's set by Github CI itself whenever things are being run by CI https://docs.github.com/en/actions/learn-github-actions/variables
I could also create a `LINERA_CI` env var, but we would have to remember to set it for every CI job we have, or figure out a way to have it always be set. Seems like a lot of extra work because of naming, but if you think it's still worth it I can do it